cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -37,6 +42,15 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
         query("SHOW USERS", assertNodesShown("User", column = "user")) {
           resultTable()
         }
+        p("This command is optionally idempotent, with the default behavior to throw an exception if the user already exists. " +
+          "Appending `IF NOT EXISTS` to the command will ensure that no exception is thrown and nothing happens should the user already exist. " +
+          "Adding `OR REPLACE` to the command will result in any existing user being deleted and a new one created.")
+        query("CREATE USER jake IF NOT EXISTS SET PASSWORD 'xyz'", ResultAssertions( r => {
+          assertStats(r, systemUpdates = 0)
+        })) {}
+        query("CREATE OR REPLACE USER jake SET PASSWORD 'xyz'", ResultAssertions( r => {
+          assertStats(r, systemUpdates = 2)
```

Comment:
It would be good to check if the visual output includes the system updates, as it does for nodesCreate in normal Cypher. I remember this was not working last week, but I hoped it was due only to a mismatch between Neo4j and docs. If it still does not work, we should report a bug for one of us to fix in docs later.

Reply:
I have not seen any visual output for the system updates no